frappe/integrations/doctype/dropbox_settings/dropbox_settings.py

```diff
@@ -90,7 +90,7 @@ def backup_to_dropbox(upload_db_backup=True):
 		dropbox_settings['access_token'] = access_token['oauth2_token']
 		set_dropbox_access_token(access_token['oauth2_token'])
 
-	dropbox_client = dropbox.Dropbox(dropbox_settings['access_token'])
+	dropbox_client = dropbox.Dropbox(dropbox_settings['access_token'], timeout=None)
```

Comment:
I don't think it is sensible to keep it waiting for ever. Can you add it to wait for 100 seconds or something whichever suits your needs. Default is 30 seconds which can be used for around 150MB of backups. Is there any particular reason for setting it to None?

Reply:
it wouldn't wait forever... it would timeout when the background job times-out... we can control the timeout of the background job and not worry about dropbox's timeout right?

also, this was the [recommended solution by a dropbox guy](https://www.dropboxforum.com/t5/Dropbox-API-Support-Feedback/socket-timeout-issue-even-with-session-upload/m-p/354432/highlight/true#M20244)...